### PR TITLE
feat(ui5-wheelslider): add cyclic behaviour

### PR DESCRIPTION
**Pull request by tsanislavgatev**
**Thank you for your contribution!** 👏

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ ] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents/blob/master/CONTRIBUTING.md)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-webcomponents/blob/master/CONTRIBUTING.md#how-to-contribute) section 
- [ ] [Correct commit message style](https://github.com/SAP/ui5-webcomponents/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: packages/main/src/WheelSlider.hbs [hunk 1]
```diff
@@ -20,11 +21,11 @@
 	<div id="{{this._id}}--inner" class="ui5-wheelslider-inner">
 	<div id="{{this._id}}--selection-frame" class="ui5-wheelslider-selection-frame">
 		</div>
-		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper" @wheel="{{_handleWheel}}">
+		<div id="{{this._id}}--wrapper" class="ui5-wheelslider-wrapper">
 			{{#if _expanded}}
 				<ul id="{{this._id}}--items-list">
-					{{#each items}}
-						<li class="ui5-wheelslider-item">{{this}}</li>
+					{{#each showItems}}
```
Comment (vladitasev) [inline packages/main/src/WheelSlider.hbs]: this sounds like a boolean. Should be "itemsToShow", "displayedItems", "allItems" etc...

File: packages/main/src/WheelSlider.js [hunk 2]
```diff
@@ -67,6 +67,10 @@ const metadata = {
 		_items: {
 			type: Object,
 		},
+
+		cyclic: {
+			type: Boolean,
```
Comment (vladitasev) [inline packages/main/src/WheelSlider.js]: Add jsdoc for this property, it is public

File: packages/main/src/WheelSlider.js [hunk 3]
```diff
@@ -163,10 +172,40 @@ class WheelSlider extends UI5Element {
...
+	_itemIndex(id) {
+		return Number(id.replace(/^\D+/g, ""));
+	}
+
+	_timesMultipliedOnCyclic() {
+		const minElementsInCyclicWheelSlider = 70;
+		const repetition = Math.round(minElementsInCyclicWheelSlider / this._items.length);
+		const minRepetitionCount = 3;
+
+		if (repetition < minRepetitionCount) {
```
Comment (vladitasev) [inline packages/main/src/WheelSlider.js]: Math.min is shorter, please use it

File: packages/main/src/WheelSlider.js [hunk 4]
```diff
@@ -163,10 +172,40 @@ class WheelSlider extends UI5Element {
...
+		}
+
+		return repetition;
+	}
+
 	get items() {
 		return this._items || [];
 	}
 
+	get showItems() {
```
Comment (vladitasev) [inline packages/main/src/WheelSlider.js]: Getters are supposed to be pure functions. It's an anti-pattern to change state in a getter. You should either have a getter, or _showItems, but not both. Please think of an elegant way to remove one or the other.

File: packages/main/src/WheelSlider.hbs [hunk 5]
```diff
@@ -20,11 +21,11 @@
...
+						<li class="ui5-wheelslider-item" id="item--{{@index}}">{{this}}</li>
```
[the remark this answers is not in the document]
Comment (tsanislavgatev) [reply packages/main/src/WheelSlider.hbs]: Done

File: packages/main/src/WheelSlider.js (hunk 3 above)
Comment (tsanislavgatev) [reply packages/main/src/WheelSlider.js]: Done

File: packages/main/src/WheelSlider.js (hunk 2 above)
Comment (tsanislavgatev) [reply packages/main/src/WheelSlider.js]: Done

File: packages/main/src/WheelSlider.hbs (hunk 1 above)
Comment (tsanislavgatev) [reply packages/main/src/WheelSlider.hbs]: Done

File: packages/main/src/WheelSlider.js (hunk 4 above)
Comment (tsanislavgatev) [reply packages/main/src/WheelSlider.js]: Done

**Pull request closed without merging**